src/app/drive/services/download.service/downloadFile.ts

```diff
@@ -135,15 +124,19 @@ export default async function downloadFile(
   await downloadToFs(completeFilename, fileStreamPromise, support, isFirefox, abortController).catch((err) => {
     const errMessage = err instanceof Error ? err.message : (err as string);
 
-    trackFileDownloadError(userEmail, fileId, errMessage);
+    if (errMessage.includes('user aborted')) {
```

Comment:
To ensure that the abort always is caught, check use:
` if (abortController.signal.aborted) {...}`

Reply:
Added